YSE_App/galaxies/path.py

```diff
@@ -95,10 +95,15 @@ def ingest_path_results(itransient:FRBTransient,
             galaxy.redshift_err = icand.redshift_err
             galaxy.redshift_source = icand.redshift_source
             galaxy.redshift_quality = 1
-        elif hasattr(icand,'redshift_type') and icand.redshift_type == 'photo-z':
-            galaxy.photoz = icand.redshift
-            galaxy.photoz_err = icand.redshift_err
-            galaxy.photoz_source = icand.redshift_source
+
+        # Grab photo-zs from PS1
+        elif hasattr(icand, 'z_phot'):
+            galaxy.photoz = icand.z_phot
+            galaxy.photoz_err = icand.z_photErr
```

Comment:
is there error checking somewhere to insist that `z_photErr` be present if `z_phot` is present?  If not, we should probably wrap this in an `hasattr` command as well.